tests/testLinearizeLookupTable.py

```diff
@@ -203,7 +203,7 @@ def makeDetector(self, bbox=None, numAmps=None, rowInds=None, colIndOffsets=None
                 ampInfo.setName("amp %d_%d" % (i + 1, j + 1))
                 ampInfo.setBBox(boxArr[i, j])
                 ampInfo.setLinearityType(linearityType)
-                ampInfo.setLinearityCoeffs([rowInds[i, j], colIndOffsets[i, j], 0, 0])
+                ampInfo.setLinearityCoeffs(np.array([rowInds[i, j], colIndOffsets[i, j], 0, 0]))
```

Comment:
setLinearityCoeffs raised that it didn't understand the arguments. Doing this converted them to float and it worked.